src/routes/[id]/training/queues/[queueId]/data-table.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+  import {
+    createRender,
+    createTable,
+    Render,
+    Subscribe,
+  } from "svelte-headless-table";
+  import { readable } from "svelte/store";
+  import DataTableActions from "./data-table-actions.svelte";
+  import * as Table from "$lib/components/ui/table";
+  import { addPagination } from "svelte-headless-table/plugins";
+  import { Button } from "$lib/components/ui/button";
+  import type { TrainingQueueMembership } from "@prisma/client";
+
+  export let members: TrainingQueueMembership[];
+
+  const timeString = (joinedAt: Date): string => {
```

Comment:
We have a similar function in `src/routes/admin/connections/+page.svelte`. I think we should just merge them into a single implementation and move it to $lib.

Reply:
Have done this! 278ea687720e8c0371c2e326582ecad9177137f9